[2024-07-29T16:49:41.161+0900] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-29T16:49:41.171+0900] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: movie.get_data scheduled__2024-07-27T04:10:00+00:00 [queued]>
[2024-07-29T16:49:41.174+0900] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: movie.get_data scheduled__2024-07-27T04:10:00+00:00 [queued]>
[2024-07-29T16:49:41.174+0900] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-07-29T16:49:41.180+0900] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): get_data> on 2024-07-27 04:10:00+00:00
[2024-07-29T16:49:41.183+0900] {standard_task_runner.py:64} INFO - Started process 10143 to run task
[2024-07-29T16:49:41.186+0900] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'movie', 'get_data', 'scheduled__2024-07-27T04:10:00+00:00', '--job-id', '583', '--raw', '--subdir', 'DAGS_FOLDER/movie.py', '--cfg-path', '/var/folders/qv/0knwm5h50ml0skpppwpbks_r0000gp/T/tmpcvv5x3ch']
[2024-07-29T16:49:41.187+0900] {standard_task_runner.py:91} INFO - Job 583: Subtask get_data
[2024-07-29T16:49:41.209+0900] {task_command.py:426} INFO - Running <TaskInstance: movie.get_data scheduled__2024-07-27T04:10:00+00:00 [running]> on host jeongui-MacBookPro.local
[2024-07-29T16:49:41.234+0900] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='movie' AIRFLOW_CTX_TASK_ID='get_data' AIRFLOW_CTX_EXECUTION_DATE='2024-07-27T04:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-27T04:10:00+00:00'
[2024-07-29T16:49:41.234+0900] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-29T16:49:41.240+0900] {logging_mixin.py:188} INFO - 2024-07-27
[2024-07-29T16:49:41.240+0900] {logging_mixin.py:188} INFO - {'conf': <airflow.configuration.AirflowConfigParser object at 0x1029961d0>, 'dag': <DAG: movie>, 'dag_run': <DagRun movie @ 2024-07-27 04:10:00+00:00: scheduled__2024-07-27T04:10:00+00:00, state:running, queued_at: 2024-07-29 07:49:02.640602+00:00. externally triggered: False>, 'data_interval_end': DateTime(2024, 7, 28, 4, 10, 0, tzinfo=Timezone('UTC')), 'data_interval_start': DateTime(2024, 7, 27, 4, 10, 0, tzinfo=Timezone('UTC')), 'ds_nodash': '20240727', 'execution_date': <Proxy at 0x114adf8c0 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x114b1dbc0>, 'execution_date', DateTime(2024, 7, 27, 4, 10, 0, tzinfo=Timezone('UTC')))>, 'expanded_ti_count': None, 'inlets': [], 'logical_date': DateTime(2024, 7, 27, 4, 10, 0, tzinfo=Timezone('UTC')), 'macros': <module 'airflow.macros' from '/Users/DE32/.pyenv/versions/3.11.9/envs/note/lib/python3.11/site-packages/airflow/macros/__init__.py'>, 'map_index_template': None, 'next_ds': <Proxy at 0x114b86b80 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x114b1dbc0>, 'next_ds', '2024-07-28')>, 'next_ds_nodash': <Proxy at 0x114b86600 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x114b1dbc0>, 'next_ds_nodash', '20240728')>, 'next_execution_date': <Proxy at 0x114b665c0 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x114b1dbc0>, 'next_execution_date', DateTime(2024, 7, 28, 4, 10, 0, tzinfo=Timezone('UTC')))>, 'outlets': [], 'params': {}, 'prev_data_interval_start_success': None, 'prev_data_interval_end_success': None, 'prev_ds': <Proxy at 0x114b65d00 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x114b1dbc0>, 'prev_ds', '2024-07-26')>, 'prev_ds_nodash': <Proxy at 0x114bc9380 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x114b1dbc0>, 'prev_ds_nodash', '20240726')>, 'prev_execution_date': <Proxy at 0x114bc9440 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x114b1dbc0>, 'prev_execution_date', DateTime(2024, 7, 26, 4, 10, 0, tzinfo=Timezone('UTC')))>, 'prev_execution_date_success': <Proxy at 0x114bc9500 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x114b1dbc0>, 'prev_execution_date_success', None)>, 'prev_start_date_success': None, 'prev_end_date_success': None, 'run_id': 'scheduled__2024-07-27T04:10:00+00:00', 'task': <Task(PythonOperator): get_data>, 'task_instance': <TaskInstance: movie.get_data scheduled__2024-07-27T04:10:00+00:00 [running]>, 'task_instance_key_str': 'movie__get_data__20240727', 'test_mode': False, 'ti': <TaskInstance: movie.get_data scheduled__2024-07-27T04:10:00+00:00 [running]>, 'tomorrow_ds': <Proxy at 0x114bc95c0 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x114b1dbc0>, 'tomorrow_ds', '2024-07-28')>, 'tomorrow_ds_nodash': <Proxy at 0x114bc9680 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x114b1dbc0>, 'tomorrow_ds_nodash', '20240728')>, 'triggering_dataset_events': <Proxy at 0x114b28380 with factory <function _get_template_context.<locals>.get_triggering_events at 0x114ad77e0>>, 'ts': '2024-07-27T04:10:00+00:00', 'ts_nodash': '20240727T041000', 'ts_nodash_with_tz': '20240727T041000+0000', 'var': {'json': None, 'value': None}, 'conn': None, 'yesterday_ds': <Proxy at 0x114bc9740 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x114b1dbc0>, 'yesterday_ds', '2024-07-26')>, 'yesterday_ds_nodash': <Proxy at 0x114bc9800 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x114b1dbc0>, 'yesterday_ds_nodash', '20240726')>, 'templates_dict': None}
[2024-07-29T16:49:41.240+0900] {logging_mixin.py:188} INFO - ====================
[2024-07-29T16:49:41.240+0900] {logging_mixin.py:188} INFO - ds_nodash => 20240727
[2024-07-29T16:49:41.240+0900] {logging_mixin.py:188} INFO - kwargs type => <class 'dict'>
[2024-07-29T16:49:41.241+0900] {logging_mixin.py:188} INFO - ====================
[2024-07-29T16:49:41.242+0900] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-29T16:49:41.242+0900] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/Users/DE32/.pyenv/versions/3.11.9/envs/note/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/DE32/.pyenv/versions/3.11.9/envs/note/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/DE32/.pyenv/versions/3.11.9/envs/note/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/DE32/.pyenv/versions/3.11.9/envs/note/lib/python3.11/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/DE32/.pyenv/versions/3.11.9/envs/note/lib/python3.11/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/DE32/airflow/dags/movie.py", line 32, in get_data
    from movie.api.call import get_key, save2df
ModuleNotFoundError: No module named 'movie.api'; 'movie' is not a package
[2024-07-29T16:49:41.244+0900] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=movie, task_id=get_data, run_id=scheduled__2024-07-27T04:10:00+00:00, execution_date=20240727T041000, start_date=20240729T074941, end_date=20240729T074941
[2024-07-29T16:49:41.248+0900] {standard_task_runner.py:110} ERROR - Failed to execute job 583 for task get_data (No module named 'movie.api'; 'movie' is not a package; 10143)
[2024-07-29T16:49:41.291+0900] {local_task_job_runner.py:243} INFO - Task exited with return code 1
[2024-07-29T16:49:41.299+0900] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-29T16:49:41.300+0900] {local_task_job_runner.py:222} INFO - ::endgroup::
